style-sheets/primevue-components-overrides.scss

```diff
@@ -377,35 +378,64 @@ label {
 .#{variables.$primevue-prefix}-card {
     box-shadow: none;
     border: 0.0625rem solid map.get(lightTheme.$light-theme, 'border-subtle-00');
-    padding: 2rem;
+    width: 38rem;
```

Comment:
Hi Jota, we don't put width in common repo, cause we might want it to have a different width in other project. Let's put width in our local.

Reply:
My bad, I'll fix it. Thanks!